seeds/transfers/transfers_bnb_agg_day_v2_sample.csv

```diff
@@ -0,0 +1,25 @@
+blockchain,day,wallet_address,token_address,symbol,unique_transfer_id,amount_raw,amount
```

Comment:
You have included a seed file which is a nice touch. However, it seems like you don't run any dbt tests against it. 

My suggestion would be to either setup a test using this [generic test file](https://github.com/duneanalytics/spellbook/blob/49964675327d46400255272fd11dffe7f79365b3/tests/generic/compare_column_values_to_seed_values.sql) (it's similar approach to DEX trades token bought test but more generic in that you can define the column) or remove this seed file.

If you setup a test, you will need to add this seed to `dbt_project.yml` file. :)

Reply:
Currently I managed to setup tests for bnb transfers aggregator. However my tests may still need to improve in near future, as I am still not very experienced with dbt test at the moment. There are some things I may need to discuss about the test:

1. Currently my tests can only be put into `tests/generic/` directory so it can be recognized by dbt, originally I want to put my tests in `tests/transfers_bnb`. I think my tests make the current directory a bit messy.
2. My naming for the test may not be proper.